samples/bluetooth/iso_combined_bis_and_cis/boards/nrf54lv10dk_nrf54lv10a_cpuapp.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_RX_STACK_SIZE=2048
```

Comment:
Why is it needed to increase the stack size? Is the default stack size too small? If yes, it sounds like the default should be changed

Reply:
IMO we should then update the default instead of applying an overlay to every single sample for every single board